qdax/core/emitters/multi_emitter.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional, Tuple
+
+import jax
+from jax import numpy as jnp
+
+from qdax.core.containers.repertoire import Repertoire
+from qdax.core.emitters.emitter import Emitter, EmitterState
+from qdax.types import Descriptor, ExtraScores, Fitness, Genotype, RNGKey
+
+
+class MultiEmitterState(EmitterState):
+    emitter_states: Tuple[EmitterState, ...]
+
+
+class MultiEmitter(Emitter):
+    def __init__(self, emitters: Tuple[Emitter, ...]):
+        self.emitters = emitters
+
+    def emit(
+        self,
+        repertoire: Optional[Repertoire],
+        emitter_state: Optional[MultiEmitterState],
+        random_key: RNGKey,
+    ) -> Tuple[Genotype, RNGKey]:
+        assert emitter_state is not None
+        assert len(emitter_state.emitter_states) == len(self.emitters)
+
+        random_key, subkey = jax.random.split(random_key)
+        subkeys = jax.random.split(subkey, len(self.emitters))
+
+        all_offsprings = []
+        for emitter, sub_emitter_state, _key_emitter in zip(
```

Comment:
What do you think about replacing `_key_emitter` by `subkey`.

Reply:
I agree